### ISSUE TITLE
any of the `plyranges` "good-first-issue" issues

### ISSUE THREAD

**Issue by mikelove** [tidyomics/genomics-todos#3]
See here:

https://github.com/sa-lee/plyranges/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue

**Comment by KittJonathan** [comment]
Hi ! I could start by working on the plyranges vignette figures

**Comment by nahid18** [comment]
Can anyone provide a PDF version of this: [common-tasks.Rmd](https://github.com/sa-lee/plyrangesWorkshops/blob/master/vignettes/common-tasks.Rmd)? Hopefully I can work on the [plyranges cheatsheet](https://github.com/sa-lee/plyranges/issues/26).

I tried to do it with `knittr` but having trouble installing `plyranges`

**Comment by mikelove** [comment]
> having trouble installing plyranges

Want to make a support.bioconductor.org post and I'll help out? Post your installation error etc.

**Comment by nahid18** [comment]
@mikelove I fixed the installation issue.

But trying to create the PDF from the .Rmd file above and this occurs:

![Image](https://github.com/tidybiology/tidy-ranges-tutorial/assets/43277262/ea1bef16-6a1b-4e26-9c3d-4031110c4d1d)

**Comment by mikelove** [comment]
(tip: you can copy paste code and enclose it with triple backticks on github, it will render like the following chunk)

```
this is code
```

Looks like `common-tasks.Rmd` was last updated 4 years ago. You can take a look at those lines and see if the functions need updating by cross-referencing the current man pages for plyranges.

**Comment by nahid18** [comment]
@mikelove Found a cheatsheet template from here [CONTRIBUTING.md](https://github.com/rstudio/cheatsheets/blob/main/.github/CONTRIBUTING.md) to work on the `plyranges` cheatsheet task

**Comment by KittJonathan** [comment]
@mikelove would overlap figures like these be useful for the doc ? https://bedtools.readthedocs.io/en/latest/content/tools/intersect.html

**Comment by mikelove** [comment]
@KittJonathan yes i think so! you could compute them within an Rmd using plyranges and make basic plots. I can give more pointers if you wanna start a repo.

**Comment by stemangiola** [comment]
@KittJonathan @nahid18 please add your authorship details here https://docs.google.com/spreadsheets/d/19XqhN3xAMekCJ-esAolzoWT6fttruSEermjIsrOFcoo/edit?usp=sharing

**Comment by mikelove** [comment]
just fixed access

**Comment by codons-blog** [comment]
@mikelove Okay thanks, I'll start working on that ASAP. Thought using this tool https://app.diagrams.net/

**Comment by mikelove** [comment]
Sure, you can coordinate among the other assignees here to make sure effort isn't duplicated.

**Comment by nahid18** [comment]
@codons-blog I am working on the [cheatsheet](https://github.com/sa-lee/plyranges/issues/26)
